fidesctl/src/fidesapi/main.py

```diff
@@ -49,18 +69,25 @@ async def db_action(action: DBActions) -> Dict:
     """
     action_text = "initialized"
     if action == DBActions.reset:
-        database.reset_db(config.api.database_url)
+        database.reset_db(CONFIG.api.database_url)
         action_text = DBActions.reset
-
-    configure_db()
+    configure_db(CONFIG.api.database_url)
     return {"data": {"message": f"Fidesctl database {action_text}"}}
 
 
 def start_webserver() -> None:
     "Run the webserver."
-    uvicorn.run(app, host="0.0.0.0", port=8080)
-
-
-config = get_config()
-configure_routes()
-configure_db()
+    setup_server()
+    server = Server(Config(app, host="0.0.0.0", port=8080, log_level=logging.WARNING))
+    server.run()
+
+def setup_server() -> None:
+    setup_logging(
```

Comment:
This should remain inside `start_webserver()`. It needs to be called after creating the server object, or the creation will create loggers that go unaccounted for.

Reply:
ohhh, right. will fix